scripts/post_merge_classify.py

```
@@ -0,0 +1,99 @@
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("--pr", type=int, required=True)
+    parser.add_argument("--github-comment", action="store_true")
+    args = parser.parse_args()
+
+    if shutil.which("gh") is None:
+        print("error: gh CLI not found", file=sys.stderr)
+        sys.exit(1)
+
+    r = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)